addons/udes_stock/report/deliveryslip_templates.xml

```diff
@@ -47,69 +62,38 @@
                                     <span t-field="o.u_driver_name"/>
                                 </td>
                             </tr>
-
-                            <t t-if="o.partner_id">
-                                <tr>
-                                    <th colspan="2" style="padding-top: 40px" />
-                                    <th colspan="2" style="padding-top: 40px"><strong>Customer address</strong></th>
-                                </tr>
-                                <tr>
-                                    <td colspan="2" />
-                                    <td colspan="2">
-                                        <span t-field="o.partner_id"
-                                           t-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": True}'/>
-                                    </td>
-                                </tr>
-                            </t>
-
-                            <tr>
-                                <th colspan="2" style="padding-top: 40px" t-if="o.origin">
-                                    <strong>Order Reference</strong>
-                                </th>
-                                <th colspan="2" style="padding-top: 40px" name="td_sched_date_h">
-                                    <strong>Date</strong>
-                                </th>
-                            </tr>
-                            <tr>
-                                <td colspan="2">
-                                    <t t-foreach="set(o.mapped('u_first_picking_ids.origin'))" t-as="origin">
-                                        <strong><span t-esc="origin"/></strong>
-                                    </t>
-                                </td>
-                                <td colspan="2" name="td_sched_date">
-                                   <t t-if="o.state == 'done'">
-                                        <span t-field="o.date_done"/>
-                                   </t>
-                                   <t t-if="o.state != 'done'">
-                                        <span t-field="o.scheduled_date"/>
-                                   </t>
-                                </td>
-                            </tr>
```

Comment:
I tried in both 11 and 14 and I cannot figure out where this is used. I am removing as it is not on the reference document for the story.

Reply:
Assuming that `o` is a `stock.picking`, it should be adding things like the partner info (e.g. address). TBH I would still leave this in unless you're 100% sure it's not used and PO agree

Reply:
I have updated this to only present the picking name and the date as per discussions with PO.